### PR TITLE
Update LogParser Steps based on feedback

### PR DESCRIPTION
**Pull request by stilianouly**
### Based on the things we discussed today

- Mention commenting out of the `import Types._ contents`
- Add question about what to do in a situation where the fileContent has empty lines
- Moving the *difficult* question at the bottom as a bonus question

### CODE REVIEW

File: src/main/scala/fundamentals/level04/LogParser.scala [hunk 1]
```diff
@@ -45,7 +45,8 @@ object LogParser {
   trait LogMessage
  
  /**
-   * Once you have defined your data types, remove `import Types._` from [LogParserTest.scala](src/test/scala/fundamentals/level04/LogParserTest.scala).
+   * - Once you have defined your data types, remove `import Types._` from [LogParserTest.scala](src/test/scala/fundamentals/level04/LogParserTest.scala).
+   * - Comment out the entire contents of src/test/scala/fundamentals/level04/LogParserTest.scala
```
Comment (ssanj) [inline src/main/scala/fundamentals/level04/LogParser.scala]: should this be:

> Comment out the entire contents of src/test/scala/fundamentals/level04/Types.scala

Comment (wjlow) [reply src/main/scala/fundamentals/level04/LogParser.scala]: Yes!

Maybe do: 

```
Once you have defined your data types:
1. Remove...
2. Comment out...
```

I think numbering the actions makes it extra clear.

Thanks Stili.

Comment (stilianouly) [reply src/main/scala/fundamentals/level04/LogParser.scala]: 🤦‍♂️

File: src/main/scala/fundamentals/level04/LogParser.scala [hunk 2]
```diff
@@ -75,14 +77,9 @@ object LogParser {
...
     * = List(UnknownLog("blblbaaaaa"))
-    **/
-  def getUnknowns(logs: List[LogMessage]): List[LogMessage] = ???
-
-  /**
-    * Define a function that returns the error log with the most recent (highest) timestamp.
     *
-    * What if we cannot find any error logs in the input?
-    */
+    * What if we get an empty line from the fileContent?
```
Comment (wjlow) [inline src/main/scala/fundamentals/level04/LogParser.scala]: This is good, but what you removed is valid too.

If there are no error logs, e.g. you have a List of Info logs, you want to return None.

**Comment by wjlow** [comment]
@stilianouly aside from that one comment I added, this is good to merge!

**Pull request merged**